egcg_core/clarity.py

```diff
@@ -87,6 +87,16 @@ def get_species_from_sample(sample_name):
             return get_species_name(species_string)
 
 
+def get_genome_version(dataset_name, species):
```

Comment:
Could you keep the same function signature (same parameters and same return) as before to ensure backward compatibility. You can make the species an optional argument.
Also `dataset_name` is not relevant in EGCG_Core use sample_id.